pkg/registry/apis/provisioning/jobs/watchset.go

```diff
@@ -0,0 +1,379 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/tilt-dev/tilt-apiserver/blob/main/pkg/storage/filepath/watchset.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Kubernetes Authors.
+
+// See also
+// https://github.com/grafana/grafana/blob/v11.1.9/pkg/apiserver/storage/file/watchset.go
```

Comment:
Copied from an earlier storage implementation.  When/if we back the job-queue with unified storage, this will not be necessary